### ISSUE TITLE
Add integration with fixie framework

### ISSUE THREAD

**Issue by Suremaker** [LightBDD/LightBDD#67]
#### Description
* Check how mature is the framework,
* Conribute to it to solve any showstoppers for integrating it with LightBDD,
* Add integration package
#### Progress

- [ ] Feature is implemented,
- [ ] Ensured backward-compatibility,
- [ ] Ensured good debugging experience

**Comment by Suremaker** [comment]
LightBDD requires an ability to execute code before and after all tests are executed, which fixie does not offer currently (https://github.com/fixie/fixie/issues/123)

**Comment by Suremaker** [comment]
The LightBdd.Fixie2 package has been added and will become available with LightBDD 2.4.0

Requirements to run LightBDD tests with Fixie:
1. Include LightBDD.Fixie2 package
2. Include Fixie package (for VS test discovery) and/or `<DotNetCliToolReference Include="Fixie.Console" Version="2.0.0" />` to run `dotnet fixie`
3. Add following classes to test project:
    ```c#
    class ConfiguredLightBddScope : LightBddScope { }
    class WithLightBddDiscovery : LightBddDiscoveryConvention { }
    ```

Implemented features:
* feature classes have to derive from `FeatureFixture`,
* scenario methods have to have `[Scenario]` attribute,
* scenario methods can be parameterized, where parameters are passed with `[InlineCase("param1", 5, ...)]`,
* it is possible to run `dotnet fixie -- category:Cat1 category:Cat2 ...` to run only scenarios with `[ScenarioCategory("Cat1")]` attributes applied on scenario method or fixture class,
* LightBDD behaviour can be configured by overriding `OnConfigure()` method in `class ConfiguredLightBddScope: LightBddScope { }`
* scenarios can be ignored with `[IgnoreScenario()]` attribute or `StepExecution.Current.IgnoreScenario()` method call.

Execution behavior:
* all scenarios are executed with **new** instance of fixture class (similarly to xunit / mstest) which offers the best isolation of tests
